serialization.rst

```diff
@@ -350,6 +414,16 @@ iterate over the array and set the properties without modifications. It
     { "foo": "yes", "__pclass": { "$type": "80", "$binary": "MyClass" } }
       -> stdClass { $foo => "yes", "__pclass" => Binary(0x80, "MyClass") }
 
+::
+
+    /* typemap: [ 'types' => [ 'UTCDateTime' => 'UTCDateTimeWrapper' ] ] */
+    { "date" : ISODate("2016-07-19T16:49:54") }
+      -> stdClass { $date => UTCDateTimeWrapper(…) }
+
+    /* typemap: [ 'types' => [ 'UTCDateTime' => 'UTCDateTimeAsUnixTimestamp' ] ] */
```

Comment:
This looks like the example I'd consider adding to the paragraph in https://github.com/mongodb/mongo-hhvm-driver/pull/111/files#r72651182.